### PR TITLE
Choosing between Django Flask and FastAPI New branch

### PR DESCRIPTION
**Pull request by jamessandy**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by jamessandy** [comment]
This Fixes #500

**Comment by jamessandy** [comment]
@lalith1403 done

**Comment by lalith1403** [comment]
@hectorkambow This is a new PR for an older article.

**Comment by hectorkambow** [comment]
#734

File: articles/Choosing-between-Django-Flask-and-FastAPI/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Django has the biggest community because of its wide use and popularity next to the Flask which also has a thriving community. FastAPI on the other hand has a small community because it's fairly new.
+
+**Performance**
+In performance, FastAPI leads because it is speed-oriented then next to Flask and finally Django which is not very fast.
+
+**Flexibility**
+Flexibility is something developers value a lot, and Flask is more flexible than Django. Fast API, on the other hand is flexible code-wise and doesn't restrict the code layout. So we can say Flask is the most flexible among all of the above.
+
+**Job openings**
+No doubt they are more job openings in the python web ecosystem with Django requirement next to Flask and finally Fast API which is few, so if your is aim to get a job quickly then Django should be your go-to guy.
```
Comment (lalith1403) [inline articles/Choosing-between-Django-Flask-and-FastAPI/index.md:93]: no doubt there are. I would suggest checking for such minor errors throughout the article to receive the complete payout. I believe another round of review on this article will result in a tiered payout.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Check for grammatical errors thoroughly.

**Comment by jamessandy** [comment]
I have done the final review, you an please go ahead and review.

**Comment by hectorkambow** [comment]
@jamessandy  
Please be sure to go over the final article to include any applicable change to future articles. 

Also be sure to use the resources we have listed out in our Resources Page - such as the basic [proofreading services ](https://www.fiverr.com/johnbatsoniii/proofread-and-edit-your-writing-for-publication?context_referrer=user_page&ref_ctx_id=9caf75ed-ce79-4880-b2fc-5b09e92e23d6&pckg_id=1&pos=1) to help avoid any extra changes or delays. 

We tend to work on polished articles 1st in the queue - and any articles that require the peer reviewer to make additional changes may result in a tiered payout or PR closure. 🚀

**Pull request merged**